rpcd/playbooks/roles/beaver/tasks/beaver_install.yml

```diff
@@ -50,6 +50,9 @@
     - beaver-install
     - beaver-init-file
 
+- name: Create runlevel links
+  command: update-rc.d -f beaver defaults
+
```

Comment:
is it not possible to do something like this https://github.com/rcbops/rpc-openstack/pull/1275/commits/a335fa383642787a866b3ee23c12db41453ffb35